regression-tests/18-compile-arm-ports/Makefile

```diff
@@ -54,6 +54,8 @@ sensniff/ev-aducrf101mkxz \
 cfs-coffee/cc2538dk \
 cfs-coffee/openmote-cc2538 \
 cfs-coffee/zoul \
+fat/zoul:BOARD=remote-reva \
```

Comment:
The `remote-reva` doesn't support the micro-SD enable/disable, but the `remote-revb` should be fully working, officially I would prefer to support the `remote-revb` only